Core/Contributions/Solutions Software/SSW Widget Enhancements.pax

```diff
@@ -303,6 +346,41 @@ resendTo: aView
 
 !PushButton methodsFor!
 
+alignment
+
+	| style |
+
+	style := self baseStyle bitAnd: BS_CENTER.
+
+	style = BS_LEFT ifTrue: [^#left].
+	style = BS_CENTER ifTrue: [^#center].
+	style = BS_RIGHT ifTrue: [^#right].
+
+	^#center!
+
+alignment: aSymbol
+
+	| mask |
+
+	self baseStyleMask: BS_CENTER set: false.
```

Comment:
This might end up creating the control twice as it is changing the base style twice. It looks like this could use View>>baseStyle:maskedBy: (e.g. as in TextEdit>>setAlignment:)? I realise it is private, but it shouldn't be.